docs/00-authors.md

```diff
@@ -3,29 +3,38 @@
 
 ## awesome - awesome window manager
 
-**Author**:
+**Maintainer**:
 
- * © 2007-2009 Julien Danjou &lt;julien@danjou.info>
+ * © 2009- Uli Schlachter &lt;psychon@znc.in>
 
-**Contributors**:
+**Main contributors**:
 
- * © 2007-2009 Nikos Ntarmos &lt;ntarmos@ceid.upatras.gr>
- * © 2007-2009 Arnaud Fontaine &lt;arnau@debian.org>
- * © 2008-2009 Damien Leone &lt;damien.leone@gmail.com>
- * © 2008-2009 Gregor Best &lt;farhaven@googlemail.com>
- * © 2008-2009 Maarten Maathuis &lt;madman2003@gmail.com>
- * © 2008-2009 koniu &lt;gkusnierz@gmail.com>
- * © 2009 Uli Schlachter &lt;psychon@znc.in>
+ * © 2013- Emmanuel Lepage-Vallee &lt;elv1313@gmail.com>
+ * © 2014- Daniel Hahler &lt;git@thequod.de>
```

Comment:
I would rather not have my email in there. But a link to GitHub, Twitter, or Daniel.Hahler.de instead.